### PR TITLE
Some useful commit message

### PR DESCRIPTION
**Pull request by rchaks**
### Description
- Added some useful information

### Testing
- Tested feature x by doing y

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,4 @@
-# gkat-test
\ No newline at end of file
+# gkat-test
+
+This is some useful information
```
Comment (gkatsikis) [inline README.md:3]: this was not pretty damn useful

Comment (gkatsikis) [reply README.md:3]: I prefer more useful things next time

**Review by gkatsikis — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (rchaks) [reply README.md:3]: Fixed it and added useful info

**Review by gkatsikis — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (gkatsikis) [reply README.md:3]: gracias

**Pull request merged**